src/pages/Ranking/index.js

```diff
@@ -24,94 +18,35 @@ const Ranking = () => {
     goToUsersStatistics,
   } = useRankingData();
 
+  const tabsConfig = [
+    {
+      label: `Today (${dailyResults.length})`,
+      content: (
+        <Today
+          dailyResults={dailyResults}
```

Comment:
Instead of passing this as a prop to the component, WDYT about taking out the data from the useRankingData on Today component? This could be as well applied to AllTime component.